src/errors.cpp

```diff
@@ -102,13 +96,11 @@ class dispatch_category_t:
         if(code == cocaine::error::dispatch_errors::uncaught_error)
             return "uncaught invocation exception";
 
-        return "cocaine.rpc.dispatch error";
+        return cocaine::format("generic %s error", name());
```

Comment:
C++ does it more verbosely: `::format("%s %d", name(), code);` which result in `cocaine.rpc.dispatch: 42`.